services/docker-compose.yml

```diff
@@ -14,8 +14,8 @@ services:
     hostname: "{{.Node.Hostname}}-{{.Task.Slot}}"
     environment:
       API_SERVER_DEV_FEATURES_ENABLED: ${API_SERVER_DEV_FEATURES_ENABLED}
-      API_SERVER_DEV_HTTP_CALLS_LOGS_PATH: ${API_SERVER_DEV_HTTP_CALLS_LOGS_PATH}
```

Comment:
should this also be removed from the `repo.config`s?

Reply:
it is also removed. See link devops section